gp-templates/project-delete.php

```diff
@@ -0,0 +1,16 @@
+<?php
+gp_title( sprintf( __( 'Delete Project %s &lt; GlotPress', 'glotpress' ),  $project->name ) );
+gp_breadcrumb_project( $project );
+gp_tmpl_header();
+?>
+<h2><?php echo wptexturize( sprintf( __( 'Delete project "%s"', 'glotpress' ), esc_html( $project->name ) ) ); ?></h2>
```

Comment:
To avoid `wptexturize()` you add the correct quotes to the string: `'Delete project &#8220;%s&#8221;'`.